Open-Source/Contribute/Readme.md

```diff
@@ -8,7 +8,8 @@
 - [Anon-chat-app](https://github.com/Dun-sin/anon-chat-app) 
 - [AOSSIE](https://summerofcode.withgoogle.com/archive/2020/organizations/5037209255673856)
 - [appwrite](https://github.com/appwrite/appwrite) 
-- [awesome-github-profiles](https://github.com/EddieHubCommunity/awesome-github-profiles) 
+- [awesome-github-profiles](https://github.com/EddieHubCommunity/awesome-github-profiles)
+- [Amplication](https://github.com/amplication) 
```

Comment:
Please add the name in alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient